src/Phansible/Resources/ansible/roles/xdebug/tasks/main.yml

```diff
@@ -2,3 +2,11 @@
 - name: Install xDebug
   sudo: yes
   apt: pkg=php5-xdebug state=latest
+
+- name: Apply xDebug settings
+  sudo: true
+  lineinfile: dest=/etc/php5/mods-available/xdebug.ini
```

Comment:
Maybe a better way to do this will be: http://docs.ansible.com/ansible/ini_file_module.html

It will remove comments of the file. But that's not a major problem.